### PR TITLE
refactor(torii): queries for them to not block

### PR DESCRIPTION
**Pull request by lambda-0x**
**Stack**:
- #2391 ⬅
- #2385

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: crates/torii/core/src/sql.rs [hunk 1]
```diff
@@ -316,18 +320,18 @@ impl Sql {
...
-        .bind(event_id)
-        .bind(entity_id)
-        .fetch_one(&self.pool)
-        .await?;
+        let update_query = "UPDATE entities SET updated_at=CURRENT_TIMESTAMP, executed_at=?, \
+                            event_id=? WHERE id = ? RETURNING *";
 
-        update_entity.updated_model = Some(wrapped_ty);
-        self.query_queue.push_publish(BrokerMessage::EntityUpdated(update_entity));
+        self.query_queue.push_publish_query(
```
Comment (Larkooo) [inline crates/torii/core/src/sql.rs:326]: We should ensure that set_model_member also works when parallelized

File: crates/torii/libp2p/src/server/mod.rs [hunk 2]
```diff
@@ -291,6 +292,7 @@ impl<P: Provider + Sync> Relay<P> {
                                         peer_id = %peer_id,
                                         "Message set."
                                     );
+                                    self.db.execute().await.unwrap();
```
Comment (Larkooo) [inline crates/torii/libp2p/src/server/mod.rs:295]: We should also execute the queries for the set_entity that is further down. Can we refactor the logic here to reuse the same self.db.execute() everywhere? We can add a function that sets the entity and directrly executes the db queries

**Comment by lambda-0x** [comment]
while trying to test syncing the flippyflop world noticed that we are spending alot of time for poseidon hash calculation. will have to add a cache for it. 
Screenshot 2024-09-06 at 16 39 16@2x

File: crates/torii/core/src/sql.rs (hunk 1 above)
Comment (lambda-0x) [reply crates/torii/core/src/sql.rs:326]: any reason it wouldn't work?

**Comment by lambda-0x** [comment]
closing in favour of #2397

**Pull request closed without merging**